src/app/static/templates/404.ftl

```diff
@@ -16,15 +16,15 @@
 </div>
 <hr>
 <div class="d-flex justify-content-center">
-    <a href="/"><img src="public/images/unaids_logo.png" class="large-logo mt-5"/></a>
+    <a href="/"><img src="/public/images/unaids_logo.png" class="large-logo mt-5"/></a>
 </div>
 
 <div id="partner-logos" class="logos d-flex justify-content-center mt-5">
-    <a href="https://www.fjelltopp.org"><img src="public/images/fjelltopp_logo.png" class="small-logo"></a>
```

Comment:
This was broken on production too, you can check by going to a non-existent root `naomi.unaids.org/not/a/path`

Reply:
Good catch, thanks. Have updated that in this template and the login template.